build/rubygem-simp-cli.spec

```diff
@@ -12,16 +12,42 @@
 Summary: a cli interface to configure/manage SIMP
 Name: rubygem-%{gemname}
 Version: %{cli_version}
-Release: 0
+Release: 0%{?dist}
 Group: Development/Languages
 License: Apache-2.0
 URL: https://github.com/simp/rubygem-simp-cli
 Source0: %{name}-%{cli_version}-%{release}.tar.gz
 Source1: %{gemname}-%{cli_version}.gem
-Requires: puppet >= 3
-Requires: facter >= 2.2
-Requires: rubygem(%{gemname}-highline) >= %{highline_version}
+Requires: cracklib
+Requires: createrepo
+Requires: curl
+Requires: diffutils
+Requires: elinks
+Requires: facter >= 3
+Requires: grep
+Requires: iproute
+Requires: net-tools
+Requires: policycoreutils
 Requires: pupmod-herculesteam-augeasproviders_grub >= 3.0.1
+Requires: pupmod-simp-network >= 6.0.3
+Requires: pupmod-simp-resolv >= 0.1.1
+Requires: pupmod-simp-simplib >= 3.11.1
+Requires: puppet >= 5
+Requires: puppetserver >= 5
```

Comment:
@lnemsick-simp I see why this is here, but I don't think that we want it. I know that a common use case is to install this and then run `simp config` just to get an output file to apply to other systems. In the case of the `bootstrap` operation, I think that we should probably just let it error out (which I think it does properly) if you're not on a system with a server to be configured.